buzzmobile/sense/lidar/lidar.py

```diff
@@ -2,96 +2,77 @@
 
 import cv2
 import numpy as np
-import time
 
 from cv_bridge import CvBridge, CvBridgeError
 from sensor_msgs.msg import Image
 from sensor_msgs.msg import LaserScan
+from math import sin, cos, pi
 import rospy
 
-import sys
-sys.path.append('../../include')
-import constants as c
+image_width = rospy.get_param("image_width")
+image_height = rospy.get_param("image_height")
+pixels_per_m = rospy.get_param("pixels_per_m")
+image_width_m = rospy.get_param("image_width_m")
+image_height_m = rospy.get_param("image_height_m")
+
 
-lidar_publisher = rospy.Publisher('lidar_model', Image, queue_size=0)
 
+lidar_publisher = rospy.Publisher('lidar_model', Image, queue_size=0)
 bridge = CvBridge()
 
+
 def gen_lidar_image(laser_scan):
     """ Converts a LaserScan message to a cv image, then publishes it through 
         a lidar_publisher """
     lidar_points = []
     ranges = laser_scan.ranges
-    angle = laser_scan.angle_min
-    #convert points from polar to cartesian (LaserScan origin is at (width/2, height)
+    #effectively rotate the input data by 90 degrees counterclockwise
+    angle = laser_scan.angle_min + pi / 2
+    #convert points from polar to cartesian (origin at (width/2, height))
```

Comment:
nit: Consider factoring out conversion to cartesian.

`cartesian_points = scan_to_catesian(laser_scan)` or something like that.